patches.json

```diff
@@ -21,6 +21,69 @@
       }
     }
   },
+  "MDVA-30052": {
+    "magento/magento2-base": {
+      "Customer data section invalidation logic has been improved. This release introduces a new way of invalidating all customer sections data that avoids a known issue with local storage when custom `sections.xml` invalidations are active. (Previously, private content (local storage) was not correctly populated when you had a custom *etc/frontend/sections.xml* with action invalidations).": {
+        ">=2.3.2-p2 <2.3.5": {
+          "file": "os/MDVA-30052__customer_data_section_invalidation_logic_improvement__2.3.4-p2.patch"
+        }
+      }
+    }
+  },
+  "MDVA-26694": {
+    "magento/magento2-base": {
+      "Product and catalog caches will expire as scheduled. Previously, caches expire daily, as сron runs the 'catalogrule_apply_all' task once a day which re-indexes all the catalog rules and its dependent indexers and clears the cache for all products and categories.": {
+        ">=2.3.0 <=2.3.5-p2 || 2.4.0": {
+          "file": "os/MDVA-26694__fix_product_and_catalog_cache_expiration_daily__2.3.1.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30131": {
+    "magento/magento2-base": {
+      "Code was not storing false values for custom attributes in Elastic Search, so filter was possible using 'No' for boolean attributes using layered navigation. To address that, some boolean checks have been replaces for null checks, and an array_filter call has been erased completely as no callback function was provided in it, thus always returning false as 'all entries of array equal to FALSE (see converting to boolean) will be removed'.": {
+        ">=2.3.4 <=2.3.5-p2 || 2.4.0": {
+          "file": "os/MDVA-30131__fix_storing_boolean_values_for_custom_attributes_in_Elastic_Search__2.3.4.patch"
+        }
+      }
+    }
+  },
+  "MDVA-27825": {
+    "magento/magento2-base": {
+      "Big data customer export issue. Memory leak prevented by adding clearing of Customer objects in \\Magento\\Framework\\Model\\ResourceModel\\Db\\VersionControl\\Snapshot class": {
+        ">=2.3.0 <=2.3.5-p2 || 2.4.0": {
+          "file": "os/MDVA-27825__fix_memory_leak_prevent_customer_export__2.3.3.patch"
+        }
+      }
+    }
+  },
+  "MDVA-29085": {
+    "magento/magento2-b2b-base": {
+      "After the company has been created through the API, the intended plugin will check if the company is created and dispatch the two emails accordingly.": {
+        ">=2.3.0 <=2.3.5-p1": {
```

Comment:
Versions of bundled extenstions like B2B, PageBuilder, Inventory are not the same as Magento version.
Please use https://wiki.corp.magento.com/display/SUP/Table+of+Magento+Releases for reference